chromiumcontent/BUILD.gn

```diff
@@ -360,6 +360,64 @@ if (is_electron_build && !is_component_build) {
     deps = [ ":targets" ]
   }
 
+  group("chromiumcontent_tests") {
```

Comment:
I think it would make sense to create separate groups unittests from browsertests.
Unittests are fast and it's easy to run them locally, browsertests are not that fast.